src/server/bootstrap.rs

```diff
@@ -59,15 +59,15 @@ pub async fn get_client_connection(server_directory: &Path) -> anyhow::Result<Cl
     let sd = ServerDir::open(server_directory).context("Invalid server directory")?;
     let access_record = sd.read_access_record().with_context(|| {
         format!(
-            "No running instance of HQ found (cannot read access record from {:?})",
+            "No running instance of HQ found (cannot read access record from {:?}), start server: <hq server start> or specify access record",
```

Comment:
It would be nicer to show the exact command to start the server, like:
```rust
"No running instance of HQ found (cannot read access record from {:?}). Try to start the server using `hq server start`".
```

If the server-directory does not match the default one, the example command should also contain `--server-dir <server-directory>`.